src/main/java/TaskManager.java

```diff
@@ -0,0 +1,139 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class TaskManager {
+    private static final String TODO_COMMAND = "todo";
+    private static final String DEADLINE_COMMAND = "deadline";
+    private static final String EVENT_COMMAND = "event";
+
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private int tasksCount = 0;
+
+    public void addTask(String typeOfTask, String userInput) {
+
+        try {
+            String[] inputs = userInput.split(" ");
+            String task = inputs[1];
+
+            if (typeOfTask.equalsIgnoreCase(TODO_COMMAND)) {
+                tasks.add(new ToDo(userInput.substring(userInput.indexOf(" ") + 1)));
+                System.out.println(" Got it. I've added this task: ");
+                System.out.println("   " + tasks.get(tasksCount));
+                System.out.println(" Now you have " + tasks.size() + " tasks in the list");
+                tasksCount++;
+            }
+            else if (typeOfTask.equalsIgnoreCase(DEADLINE_COMMAND)) {
+                tasks.add(new Deadline(userInput.substring(userInput.indexOf(" ") + 1, userInput.indexOf("/by")), userInput.substring(userInput.indexOf("/by") + 4)));
```

Comment:
Refactor this code into a separate function, or put in separate lines. This line is too long and is not very intuitive for understanding.